.ruby-version-next

```diff
@@ -1 +1 @@
-3.2.0
+3.2.1
```

Comment:
Since I'm in the middle of updating Node from 16 to 20. Let's not update this yet.
```suggestion
3.2.0
```

Reply:
If I understood this correctly, this will just create the ruby 3.2.1 image in ghcr.io for next time Ruby is upgrade to 3.2.1.
But this can indeed be pushed to later!

Reply:
It's because our base image versioning rely on ruby version. I want to update Node without touching Ruby versioning, so I''m going to still rely on the 3.2.0 tag but different SHA. The build also will not start unless there are changes to `Container.base` realistic we can wait til we're actually updating Ruby.